EfEnumToLookup/LookupGenerator/MetadataHandler.cs

```diff
@@ -53,15 +53,17 @@ private static IEnumerable<EnumReference> ProcessEdmProperties(IEnumerable<EdmPr
 			// get mapped table name from mapping, or fall-back to just the name if no mapping is set,
 			// I have no idea what causes Table to be null, and I have no unit test for it yet, but I have seen it.
 			var table = mappingFragment.StoreEntitySet.Table ?? mappingFragment.StoreEntitySet.Name;
-
-			foreach (var edmProperty in properties)
+            var schema = mappingFragment.StoreEntitySet.Schema ?? "dbo";

```

Comment:
incorrect indentation - this project uses tabs. Think there's an editorconfig file, if not there should be. http://editorconfig.org/